### PR TITLE
Update/readme

### PR DESCRIPTION
**Pull request by AHReccese**
#### Reference Issues/PRs
Update `README.md`

### CODE REVIEW

**Comment by AHReccese** [comment]
@sepandhaghighi
Please check whether or not I need to add more details about this release works somewhere else.

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - PyMilo exception types added in `pymilo/exceptions/__init__.py`
 - PyMilo exception types added in `pymilo/__init__.py`
 ### Changed
+- `README.md` updated
```
Comment (sepandhaghighi) [inline CHANGELOG.md:27]: Duplication (line `30`)

File: README.md [hunk 2]
```diff
@@ -157,27 +157,37 @@ You can easily serve your ML model from a remote server using `ML streaming` fea
...
 #### Server
-Let's assume you are in the remote server and you want to import the exported JSON file and start serving your model!
+Let's assume you are in the remote server and you want to import the exported JSON file and start serving your model through `REST` protocol!
 ```pycon
 >>> from pymilo import Import
->>> from pymilo.streaming import PymiloServer
+>>> from pymilo.streaming import PymiloServer, CommunicationProtocol
 >>> my_model = Import("model.json").to_model()
->>> communicator = PymiloServer(model=my_model, port=8000).communicator
+>>> communicator = PymiloServer(
```
Comment (sepandhaghighi) [inline README.md:168]: Please avoid splitting lines when using `pycon` (Python Console) mode

File: README.md [hunk 3]
```diff
@@ -157,27 +157,37 @@ You can easily serve your ML model from a remote server using `ML streaming` fea
...
 ```
 Now `PymiloServer` runs on port `8000` and exposes REST API to `upload`, `download` and retrieve **attributes** either **data attributes** like `model._coef` or **method attributes** like `model.predict(x_test)`.
 
 #### Client
 By using `PymiloClient` you can easily connect to the remote `PymiloServer` and execute any functionalities that the given ML model has, let's say you want to run `predict` function on your remote ML model and get the result:
 ```pycon
->>> from pymilo.streaming import PymiloClient
->>> pymilo_client = PymiloClient(mode=PymiloClient.Mode.LOCAL, server_url="SERVER_URL")
+>>> from pymilo.streaming import PymiloClient, CommunicationProtocol
+>>> pymilo_client = PymiloClient(
```
Comment (sepandhaghighi) [inline README.md:181]: Please avoid splitting lines when using `pycon` (Python Console) mode

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

**Review by sepandhaghighi — APPROVED** [review]
🔥

**Pull request merged**